selection_sort/Python/selection_sort.py

```diff
@@ -0,0 +1,17 @@
+def selection_sort(arr):
+    for i in range(len(arr)):
+        min_i = min(arr[i:]) 
+        min_index = arr[i:].index(min_i) 
+        arr[i + min_index] = arr[i] 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpxipfo4mc/selection_sort/Python/selection_sort.py
+++ b/tmp/tmpxipfo4mc/selection_sort/Python/selection_sort.py
@@ -2,7 +2,7 @@
     for i in range(len(arr)):
         min_i = min(arr[i:]) 
         min_index = arr[i:].index(min_i) 
-        arr[i + min_index] = arr[i] 
+        arr[i + min_index] = arr[i]
         arr[i] = min_i
     sorted_arr = arr
     return sorted_arr
```